.github/workflows/generate-sdks.yaml

```diff
@@ -0,0 +1,27 @@
+name: Generate SDK
+on:
```

Comment:
Currently we are only generating code based on a manual workflow?

Reply:
This is called from the api update aswell so if there is a change there then we re generate